.changeset/thin-avocados-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Allow user without MANAGE_APPS to access to apps page
```

Comment:
I wish we write better changelogs, not commit messages ;)

Reply:
People will see this changelogs when browsing app history, its empty. The context is in PR description, which is hard to retrieve browsing history